platform-core-module/src/main/resources/com/softicar/platform/core/module/program-execution-style.css

```diff
@@ -1,6 +1,48 @@
+:root {
+	--PROGRAM_EXECUTION_COLOR_ERROR: #dc3545;
+	--PROGRAM_EXECUTION_COLOR_INFO: #407fbf;
+	--PROGRAM_EXECUTION_COLOR_OKAY: #4bb91e;
```

Comment:
IMO should be using our default color for these states

Reply:
ok